libfreerdp/crypto/test/TestFuzzCryptoCertificateDataMatch.c

```diff
@@ -0,0 +1,16 @@
+#include <freerdp/crypto/crypto.h>
+
+extern int LLVMFuzzerTestOneInput(const uint8_t* Data, size_t Size)
+{
+	rdpCertificate* certificate = NULL;
+	rdpCertificateStore* certificate_store = NULL;
+	rdpSettings* settings;
+
+	settings = freerdp_settings_new(0);
+	certificate_store = certificate_store_new(settings);
+	certificate_data_print(certificate_store, (rdpCertificateData*)Data);
```

Comment:
Will not work, the `rdpCertificateData` struct expects to hold `\0` terminated strings